DutchFilter/sections/specific.txt

```diff
@@ -4,6 +4,7 @@
 ! Good: example.org###rek; ||example.com/ads/
 ! Bad: example.org#@##banner_ad (for instance, should be in allowlist.txt or in antiadblock.txt)
 !
+||azurewebsites.net^$domain=npo.nl
```

Comment:
I guess it's about ad like reported here - https://github.com/AdguardTeam/AdguardFilters/issues/166205
If so, it's already fixed by a scriptlet and I cannot reproduce it, though it might be useful for platforms which do not support scriptlets.
I guess that `azurewebsites.net` might be used for other purposes than ad as well, so could this rule be more "ad related"?
For example:
```adblock
||azurewebsites.net/*adslot$domain=npo.nl
```

Reply:
The scriptlet did not work for me on Firefox with uBlock Origin (all latest version).

`azurewebsites.net` is only used for tracking and serving ads from the network connections I saw, I noticed no breakage on the site itself

Reply:
As far as I can see, `azurewebsites.net` can be used for other content as well.
Even though it doesn't break anything, I still prefer more specific rule(s), at least for now.